server/views/pages/confirmedOnReport.njk

```diff
@@ -6,35 +6,51 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/panel/macro.njk" import govukPanel %}
 
-{% set title = "Prisoner has been placed on report" %}
+{% if changed %}
+  {% set title = "Report change is confirmed" %}
+{% else %}
+  {% set title = "Prisoner has been placed on report" %}
+{% endif %}
 
 {% block pageTitle %}
-    {{ title }}
+  {{ title }}
 {% endblock %}
 
 {% block content %}
-    {{ govukPanel({
-        titleText: prisonerFirstAndLastName + ' has been placed on report',
-        html: 'Your report number is: <br><strong>' + adjudicationNumber + '</strong>',
-        classes: "govuk-!-margin-bottom-6"
-    }) }}
-
-    <h2 class="govuk-heading-m">What you must do next</h2>
-
-    {{ onReportInfo(prisonerFirstAndLastName, expirationTime, expirationDay, showPrisonerPreferredLanguage, prisonerPreferredLanguage, showPrisonerOtherLanguages, prisonerOtherLanguages, showPrisonerNeurodiversities, prisonerNeurodiversities) }}
-
-    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-
-    {{ printLink('Print a copy of this report', printHref, "left", "govuk-!-padding-bottom-0") }}
-
-    {{ govukButton({
+  <div class="govuk-grid-row">
+      {{ govukPanel({
+          titleText: bannerText,
+          html: bannerHTML,
+          classes: "govuk-!-margin-bottom-6"
+        }) }}
+
+      <h2 class="govuk-heading-m">What you must do next</h2>
+
+      <div class="confirmed-on-report-section">
+        {% if not changed %}
```

Comment:
Could we have a more descriptive name - this name indicates something has changed, but what. Maybe try `adjudicationEdited`?